go/adbc/driver/bigquery/driver.go

```diff
@@ -0,0 +1,222 @@
+package bigquery
+
+import (
+	"cloud.google.com/go/bigquery"
+	"fmt"
+	"github.com/apache/arrow-adbc/go/adbc"
+	"github.com/apache/arrow-adbc/go/adbc/driver/internal/driverbase"
+	"github.com/apache/arrow/go/v16/arrow/memory"
+	"golang.org/x/exp/maps"
+	"runtime/debug"
+	"strings"
```

Comment:
use `go fmt` to reorder and reformat these imports please. std library imports should be at the top with an empty line before any others.